addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -964,6 +971,14 @@ function factory(dependencies) {
         // Handlers
         //----------------------------------------------------------------------
 
+        /**
+         * @private
+         * @param {Event} ev
+         */
+        async _onAnyUnload(ev) {
+            this.channel && await this.channel.performRpcLeaveCall();
```

Comment:
as commented elsewhere, make an obvious condition and avoid `&&` on lines that have a side effect